core/domain/change_domain.py

```diff
@@ -222,3 +222,6 @@ def to_dict(self):
                     self, attribute_name)
 
         return base_change_dict
+
+    def __getattr__(self, name: str) -> str:
+        return self.name
```

Comment:
Actually, this is wrong, when you do `self.name` python internally calls `self.__getattr__` and that way you get into a recursion loop. You need to do `return self.__dict__[name]` I think.

Reply:
@vojtechjelinek done.